nwb_conversion_tools/datainterfaces/ecephys/axona/axonadatainterface.py

```diff
@@ -80,7 +87,6 @@ def __init__(self, filename: str):
         super().__init__(filename=filename)
 
     def get_metadata_schema(self):
-        """Compile metadata schema for the RecordingExtractor."""
         metadata_schema = super().get_metadata_schema()
         metadata_schema["properties"]["Ecephys"]["properties"].update(
             ElectricalSeries_raw=get_schema_from_hdmf_class(ElectricalSeries)
```

Comment:
actually this method is also not needed: there is no need to specify `ElectricalSeries_raw` as a metadata as this naming is done internally in the `add_electrical_series()` method. The only thing required is to specify within the `run_conversion()` is the argument : `write_as='raw/processed/lfp'`, which will write it accordingly inside the nwbfile with the correct name and description. 
Specifying this as a metadata explicitly is useful only when you need to add some custom description (which i dont see you doing on line 132)

Reply:
True, I simplified this as well. Thanks.